Polygon_mesh_processing/examples/Polygon_mesh_processing/detect_features_example.cpp

```diff
@@ -27,15 +27,25 @@ int main(int argc, char* argv[])
   typedef boost::property_map<Mesh, CGAL::edge_is_feature_t>::type EIFMap;
   typedef boost::property_map<Mesh, CGAL::face_patch_id_t<int> >::type PIMap;
   typedef boost::property_map<Mesh, CGAL::vertex_incident_patches_t<int> >::type VIMap;
+  typedef boost::property_map<Mesh, CGAL::vertex_feature_degree_t>::type VIFMap;
```

Comment:
The property map you are filling is not the vertex feature degree.
You should create a new one, similar to `edge_is_feature_t` (line 27), that describes what you are listing/detecting. I would suggest `CGAL::vertex_is_feature_t`

Reply:
It's giving an error showing no such thing as vertex_is_feature_t in CGAL.

Reply:
right
the parameter and type should be added as `edge_is_feature_t` was previously added

Reply:
How should I add it?

Reply:
I added that in the boost/graph/properties.h right now!